Source/CMakeLists.txt

```diff
@@ -13,6 +13,19 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+if(CMAKE_C_COMPILER_ID MATCHES "MSVC")
```

Comment:
Could this be if(MSVC) as seen in other files?

Reply:
Probably. I don't see any reason why not. I just kept it from the previous version.